### ISSUE TITLE
when continue from checkpoint, learning rate stays the same as before, even when set to a different value

### ISSUE THREAD

**Issue by zhuliyi0** [bghira/SimpleTuner#163]
I noticed the learning rate stays the same as previous training session if continue from a checkpoint, even if I set it to a different value. I looked at the code and found the value was probably over-written by loading scheduler state from checkpoint. Is this a must? What can be changed and what cannot?

**Comment by bghira** [comment]
the learning rate / optimizer states are loaded from the ckpt, this is simply how that works to ensure the velocity remains.

**Comment by bghira** [comment]
just wanted to let you know that when you're not using min-SNR gamma, using `--snr_weight` option can be used to augment the loss value, which is roughly the same as adjusting the learning rate.
